app/forms/developers_search_form.rb

```diff
@@ -0,0 +1,18 @@
+class DevelopersSearchForm
+  include Virtus.model
+  include ActiveModel::Model
+  include ActiveRecord::Sanitization::ClassMethods
+
+  attribute :language, String
+  attribute :programming_language, String
+
+  def search
+    return Developer.none if !language.present? && !programming_language.present?
+
+    query = Developer.includes(:developer_languages, :languages, :developer_programming_languages, :programming_languages)
+    query = query.where(languages: { code: language }) if language.present?
+    query = query.where(programming_languages: { name: programming_language }) if programming_language.present?
```

Comment:
@truongle7793 
l10, `language.present?` 
l 3, `language.present?`

You should create a method to check it

Reply:
Thanks @thienhv 
I have made the change to remove duplicate code
```
    check_language = language.present?
    check_programming_language = programming_language.present?
```